src/supported-keyframes.js

```diff
@@ -0,0 +1,14 @@
+import prefix from './prefix'
+
+/**
+ * Test if a keyframe at-rule should be prefixed or not
+ *
+ * @param {String} vendor prefix string for the current browser.
+ * @return {Boolean}
+ * @api public
+ */
+
+export default function supportedKeyframes() {
+  if (prefix.js === 'ms') return '@'
+  return `@${prefix.css}`
```

Comment:
why not `@${prefix.css}keyframes` ? so that @keyframes identifier is complete?

Reply:
It's complete now because `keyframes` it's a part of `rule.key.substr[1]` here - https://github.com/cssinjs/jss-vendor-prefixer/pull/12/files#diff-1fdf421c05c1140f6d71444ea2b27638R13